coral/src/app/features/topics/request/utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { transformAdvancedConfigEntries } from "src/app/features/topics/request/utils";
+
+describe("TopicRequest utils", () => {
+  describe("createTopicRequestPayload", () => {
+    it("should have tests", () => {
```

Comment:
Is this a block to not get a warning about an empty test suite? Looks like what I am doing in this case 😅

Reply:
🤦 . Let me add some tests, they will be pretty simple but better to have some base tests so if there are regressions, those would be easy to prove fixed with test.

Reply:
Thought it looked familiar :D

Reply:
Maybe you already know, but: 
If there are tests that I know need to be added but for some reason currently can't, I do: 

`test.todo("test case that is missing description")` 

This shows up in the test runs as a reminder, which I think it's quite nice!